### DISCUSSION TITLE
gg: anti-aliase for lines

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#11789] [Ideas]

**Comment by medvednikov** [comment]
We have that.

**Comment by ghost** [comment]
Oh, really?
How can I do that?

**Comment by medvednikov** [comment]
gg.draw_text() uses it by default

**Comment by ghost** [comment]
![dr](https://user-images.githubusercontent.com/52106048/104729108-47ac4c00-577b-11eb-9304-b60ed10f30a2.png)

gg.draw_line() does not seems on Linux.

**Comment by medvednikov** [comment]
Ok, good point. For some reason I was only associating antialiasing with text.

Lines are not antialiased.

**Comment by medvednikov** [comment]
I like the graph you built by the way.
